frontend/lib/src/declarations.d.ts

```diff
@@ -22,6 +22,8 @@ declare module "@loaders.gl/gltf"
 
 declare module "native-file-system-adapter"
 
+declare module "*.png"
```

Comment:
Why is this needed?

Reply:
I think I needed it for a reason, but that reason is no longer necessary. Will remove.